doc-Community_Collaboration_Guide/topics/contributing_to_the_framework.adoc

```diff
@@ -32,10 +32,6 @@ You must be a member of the GitHub `community-contribution-template` repository.
 
 . Click `Submit new issue`. The title and number of your open issue are displayed in a new window.
 
-.Related Information
```

Comment:
I know that @sjay72 made the decision to include links in Related Information instead of in line with the text above on purpose.  We should discuss this before making changes globally.  @pmkovar is there a reason you made the change?

Reply:
I'm in favor of adding the link to the repo where we first mention it. This improves the content usability, I think.

@sjay72 Is this supposed to follow a template?

Reply:
Rethinking this, based on what Nicole and I have been discussing and also as a result of a chat she had with Betty. My notion of "related information" is incorrect here. The link to "Issues" in this procedure isn't "related information" (i.e., not part of the steps), rather it *is* the information. I agree that having it in-line is correct.